ScoutSuite/providers/aws/authentication_strategy.py

```diff
@@ -1,17 +1,20 @@
 import boto3
 
-from ScoutSuite.providers.base.authentication_strategy import AuthenticationStrategy, AuthenticationException
+from ScoutSuite.core.console import print_error
+from ScoutSuite.providers.base.authentication_strategy import AuthenticationStrategy
+
 
 class AWSAuthenticationStrategy(AuthenticationStrategy):
     """
     Implements authentication for the AWS provider
     """
-    
+
     def authenticate(self, profile=None, **kwargs):
         session = boto3.Session(profile_name=profile)
         credentials = session.get_credentials().__dict__
 
         if credentials.get('access_key') is None:
-            raise AuthenticationException()
+            print_error('Failed to authenticate to AWS')
+            return False
```

Comment:
I think there's an inconsistency here with Azure and GCP `authenticate` methods. Both of them raise an `AuthenticationException` in case of authentication failure and don't return a `False` value...  But I don't really know if we want to return a `False` value or raise an exception, to be honest. Any opinion on the matter @Aboisier ?

Reply:
Yeah I wasn't sure either. I think returning False and printing an exception would be the best option, as it would allow `__main__.py` to exit graciously. No real reason to throw a custom exception, no?

Reply:
I'm not a fan of using a boolean to indicate success or failure when failure is not a valid use case. In this context, failure is truly exceptional, and it's important to let the caller know that it failed. The caller should handle the exception and log if it wants. The boolean solution is less verbose and could lead to problems if this method is used elsewhere.

Reply:
I think I agree with @Aboisier on this one.